lib/cli/src/commands/create_exe.rs

```diff
@@ -21,8 +21,7 @@ use wasmer_object::{emit_serialized, get_object_for_target};
 pub type PrefixerFn = Box<dyn Fn(&[u8]) -> String + Send>;
 
 const WASMER_MAIN_C_SOURCE: &[u8] = include_bytes!("wasmer_create_exe_main.c");
-#[cfg(feature = "static-artifact-create")]
-const WASMER_STATIC_MAIN_C_SOURCE: &[u8] = include_bytes!("wasmer_static_create_exe_main.c");
```

Comment:
Isn't the `wasmer_static_create_exe_main.c` file necessary for the other code / the `--object-format symbols`? If this is removed, isn't it going to impact the `create-exe-pirita` work?

Reply:
The deserialize functions for both serialized and symbol object formats now have the same signature, and they no longer require different main.c files.